<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogfileAnalyserClass</class>
 <widget class="QMainWindow" name="LogfileAnalyserClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogWitch</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExportLogfile"/>
   </widget>
   <widget class="QMenu" name="menuLogSources">
    <property name="title">
     <string>Log Sources</string>
    </property>
   </widget>   
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLogSources"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Table manipulation</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionClearLogTable"/>
   <addaction name="actionCapture"/>
  </widget>
  <widget class="QToolBar" name="toolBarFiltering">
   <property name="windowTitle">
    <string>Filtering</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddQuicksearchFilter"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open ...</string>
   </property>
  </action>
  <action name="actionOpenDummyLogger">
   <property name="text">
    <string>OpenDummyLogger</string>
   </property>
  </action>
  <action name="actionAddEntries">
   <property name="text">
    <string>AddEntries</string>
   </property>
  </action>
  <action name="actionClearLogTable">
   <property name="icon">
    <iconset>
     <normalon>:/icons/trash</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionCapture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="LogfileAnalyserResources.qrc">
     <normaloff>:/icons/active</normaloff>:/icons/active</iconset>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
   <property name="toolTip">
    <string>Capturing log entries</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionExportLogfile">
   <property name="text">
    <string>Export Logfile ...</string>
   </property>
  </action>
  <action name="actionAddQuicksearchFilter">
   <property name="icon">
    <iconset resource="LogfileAnalyserResources.qrc">
     <normaloff>:/icons/search</normaloff>:/icons/search</iconset>
   </property>
   <property name="text">
    <string>Add Quicksearch Filter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="LogfileAnalyserResources.qrc"/>
 </resources>
 <connections/>
</ui>
